Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -1192,6 +1173,22 @@
     return nil;
 }
 
+bool PDFPluginBase::populateEditorStateIfNeeded(EditorState& state) const
+{
+    if (platformPopulateEditorStateIfNeeded(state)) {
+        // Defer to platform-specific logic.
+        return true;
+    }
+
+    if (selectionString().isNull())
+        return false;
+
+    state.selectionIsNone = false;
+    state.selectionIsRange = true;
+    state.isInPlugin = true;
```

Comment:
> for something in WKContentViewInteraction

So I think that iOS-specific codepath can actually be removed altogether, since nothing sets the `isInPlugin` flag for iOS. On macOS, it's used for integration with system services (e.g. AppleScript) and Shortcuts, when obtaining selected content in the web view.

Perhaps we could just wrap this with a `PLATFORM(MAC)` compile-time flag and remove the dead code in `WKContentViewInteraction.mm`?

Reply:
Yeah I can do that if you send me a FIXME!

Reply:
Ah, I just ended up cleaning that up as a part of this PR.